src/components/Dialog/Dialog.tsx

```diff
@@ -14,6 +16,9 @@ type DialogProps = {
 export const Dialog: FC<DialogProps> = ({open, close, title, contentText, children, actions}) => {
   return (
     <MuiDialog open={open} onClose={close}>
+      <Box position="absolute" top={16} right={16}>
```

Comment:
hmm takze nejaky space vytvorit pre title, aby sa to nikdy neprekrylo? 😄 resp. prerobit tie dlhe nadpisy?